File: 3895.count-digit-appearances.java
/*
 * @lc app=leetcode id=3895 lang=java
 *
 * [3895] Count Digit Appearances
 */

// @lc code=start
class Solution {
    public int countDigitOccurrences(int[] nums, int digit) {
        int count = 0;
        for (int n : nums) {
            while (n > 0) {
                int lastdigit = n % 10;
                n = n / 10;
                if (lastdigit == digit) {
                    count++;
                }
            }
        }
        return count;
    }
}
// @lc code=end

File: 3895.count-digit-appearances.py
#
# @lc app=leetcode id=3895 lang=python
#
# [3895] Count Digit Appearances
#

# @lc code=start
class Solution(object):
    def countDigitOccurrences(self, nums, digit):
        """
        :type nums: List[int]
        :type digit: int
        :rtype: int
        """
        c = 0 
        for i in nums:
            while i > 0:
                d = i % 10
                i //= 10
                if d == digit:
                    c += 1
        return c
        
# @lc code=end

